history/kishore-papineni.md

```diff
@@ -0,0 +1,10 @@
+---
+description: Co-author of a machine translation evaluation metric
+---
+# Kishore Papineni
+Kishore Papineni is the co-author of the BLEU metric for automatic evaluation of machine translation output. 
+BLEU has been around for almost two decades and continues to be used extensively. 
```

Comment:
We'll have to update this every decade. ;-)

> In 200x, while at, he ... [BLEU](metrics/bleu.md)

The impact of BLEU can be discussed in the BLEU article, not in every co-author's article.